smartsim/entity/model.py

```diff
@@ -609,3 +627,10 @@ def _check_db_objects_colo(self) -> None:
                         "file and add it to the SmartSim Model with add_script."
                     )
                     raise SSUnsupportedError(err_msg)
+
+    def dbs_list_m(self, db_identifier):
+        self.my_list_m.append(db_identifier)
```

Comment:
I think you've renamed my_list_m, making this dead code. Also, the name of this method does not imply at all that it's appending to the list, so code written to use it won't be clear